db/migrate/20240412005358_create_documents.rb

```diff
@@ -0,0 +1,17 @@
+class CreateDocuments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :documents, id: :uuid do |t|
+      t.date :issue_date
+      t.date :due_date
+      t.decimal :amount
+      t.string :status
+      t.uuid :currency_id
+      t.uuid :company_id
+      t.uuid :document_type_id
+      t.uuid :transactions_id
+      t.uuid :financial_institutions_id
```

Comment:
the relationship must be in singular, document belongs to a single transaction
transaction_id is more appropriate
same for institutions